README.md

```diff
@@ -3,3 +3,64 @@
 Welcome to the Void documentation. This repository contains the source data
 behind <https://docs.voidlinux.org/>. Contributing to this repository follows
 the same protocol as the packages tree.
+
+## Contributing
+
+### Requirements
+
+To clone the repository and push changes you need git installed.
+
+```
+# xbps-install -S git
+```
+
+Building the Void Handbook locally requires
+[mdBook](https://rust-lang-nursery.github.io/mdBook/) to be installed. To get
+it:
+
+```
+# xbps-install -S mdBook
+```
+
+### Forking
+
+To fork the repository a [github account](https://github.com/join) is needed.
+After the account is created follow github's [handy
+guide](https://help.github.com/en/articles/fork-a-repo) on setting up a fork.
+
+Create a new
+[branch](https://help.github.com/en/articles/creating-and-deleting-branches-within-your-repository)
+to edit, and then clone the repository onto your computer and enter it:
+
+```
+$ git clone https://github.com/YOUR_USERNAME/void-docs.git
+$ cd void-docs
+```
+
+After editing the file(s), commit the changes and push them to the forked
+repository:
+
+```
+$ git add <EDITED_FILE>
+$ git commit -m "<COMMIT_MESSAGE>"
+$ git push
+```
+
+> The commit message should be in the form of `something: reduced section x and
+> moved to it's own page`
+
```

Comment:
maybe mention: All pull requests should be a single commit. If you need to change a commit you can commit the new changes with `git commit --amend` or if you do make multiple commits....